cmd/tankism/main.go

```diff
@@ -1,18 +1,54 @@
 package main
 
 import (
+	_ "embed"
 	"github.com/co0p/tankism"
+	"github.com/co0p/tankism/cmd/tankism/start"
 	"github.com/hajimehoshi/ebiten/v2"
 	"log"
 )
 
+type Client struct {
```

Comment:
moved client to cmd... this forces us to follow the import restrictions from golang. Golang does not allow import cycles. Therefore we follow the rule: 

The higher the folder the more it is allowed to import. 
Lower packages are not allowed to import from higher packages.